Source/WebCore/PAL/pal/spi/mac/NSTextInputContextSPI.h

```diff
@@ -30,19 +30,6 @@
 #import <AppKit/NSTextInputContext_Private.h>
 #import <AppKit/NSTextPlaceholder_Private.h>
 
-#if HAVE(NSTEXTPLACEHOLDER_RECTS)
-// Staging for rdar://126696059
-@interface NSTextPlaceholder (staging_126696059)
-@property (nonatomic, readonly) NSArray *rects;
-@end
-
-@protocol NSTextInputClient_Async_staging_126696059
-@optional
-- (void)insertTextPlaceholderWithSize:(CGSize)size completionHandler:(void (^)(NSTextPlaceholder *))completionHandler;
-- (void)removeTextPlaceholder:(NSTextPlaceholder *)placeholder willInsertText:(BOOL)willInsertText completionHandler:(void (^)(void))completionHandler;
-@end
-#endif // HAVE(NSTEXTPLACEHOLDER_RECTS)
-
```

Comment:
Shouldn't this be moved rather than outright deleted, to support building with the public SDK?

Reply:
fixed

Reply:
I'm not seeing the fix?

Reply:
it's in AppKitSPI.h

Reply:
and there is already

```
@interface NSTextPlaceholder : NSObject
@property (nonatomic, readonly) NSArray<NSTextSelectionRect *> *rects;
@end
```

in NSTextInputContextSPI.h